base_geoengine/static/src/js/widgets/geoengine_edit_map/field_geoengine_edit_map.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <templates xml:space="preserve">
     <t t-name="FieldGeoEngineEditMap" owl="1">
-
+        <div id="map" style="width: 100%; height: 100%;">
```

Comment:
Do we need a `style` attribute on this element? You already specify a css rule on 'o_field_geo_edit_map'. In any case, if style attributes are needed, they must come from the css and not from the style attribute on the element itself.

The value for the `id` attribute must come from the widget id to avoid duplicate if the widget is used more than one time on the same view.

Reply:
ping @kouffsamuel